src/harness/testcases/registration_testcase.py

```diff
@@ -182,7 +182,6 @@ def test_WINNF_FT_S_REG_4(self):
     request = {'registrationRequest': [device_b]}
     response = self._sas.Registration(request)['registrationResponse'][0]
     # Check registration response
-    self.assertTrue('cbsdId' in response)
```

Comment:
Actually I think we want to remove line 198.

This line can also be removed (line 188 will fail if 'cbsdId' is not in response) but it wasn't the goal of my comment.